src/test/java/com/snowflake/kafka/connector/ConnectClusterBaseIT.java

```diff
@@ -34,10 +35,13 @@ public abstract class ConnectClusterBaseIT {
 
   @BeforeAll
   public void beforeAll() {
+    Map<String, String> workerConfig = new HashMap<>();
```

Comment:
This change is caused by: https://kafka.apache.org/documentation.html#connect_plugindiscovery

The default settings was `hybrid_fail` which caused the cluster to failed on startup.